packages/opam-dune-lint/opam-dune-lint.0.1/opam

```diff
@@ -17,8 +17,8 @@ depends: [
   "ocamlfind"
   "bos"
   "fmt" {>= "0.8.7"}
-  "opam-state" {opam-version < "2.1" & < "2.1.0~~"}
-  "opam-state" {opam-version >= "2.1" & >= "2.1.0~~" & < "2.2.0~beta3"}
+  "opam-state" {>= "2.0" & < "2.1" & opam-version >= "2.0" & opam-version < "2.1"}
+  "opam-state" {>= "2.1" & < "2.2" & opam-version >= "2.1" & opam-version < "2.2"}
```

Comment:
The CI has opam version `version 2.2~beta` which is why I think we have CI errors: the constraints cannot be satisfied. Stillk, the bounds look good to me and I think we can merge.